examples/FluentUIServerSample/Pages/ComponentBindings.razor

```diff
@@ -147,15 +166,46 @@
             <label for="CheckboxValue">Checkbox Value</label>
             <FluentCheckbox Name="CheckboxValue" @bind-Value="Form.CheckboxValue" />
         </div>
+        <div style="width: 500px;">
+        <label for="SliderValue">Slider Value</label>
+        <FluentSlider id="SliderValue" Orientation="Orientation.Horizontal" Min="0" Max="100" Step="10" @bind-Value="Form.SliderValue">
+            <FluentSliderLabel Position="0">
+                0&#8451;
+            </FluentSliderLabel>
+            <FluentSliderLabel Position="10">
+                10&#8451;
+            </FluentSliderLabel>
+            <FluentSliderLabel Position="90">
+                90&#8451;
+            </FluentSliderLabel>
+            <FluentSliderLabel Position="100">
+                100&#8451;
+            </FluentSliderLabel>
+        </FluentSlider>
+    </div>
         <input type="submit" value="Send" />
         <FluentButton>Submit</FluentButton>
     </EditForm>
 </div>
 
 @code {
-    private FormModel Form { get; set; } = new();
+    public FormModel Form { get; set; } = new();
 
-    private class FormModel
+    /// <summary>
+    /// Within this constructor you can set default values for the form fields through the FormModel
+    /// </summary>
+    public ComponentBindings()
+    {
+
+        //Form.SelectValue = "Two";
+        //Form.TextFieldValue = "TextFieldValue";
+        //Form.NumberFieldValue = 123;
+        //Form.TextAreaValue = "TextAreaValue";
+        //Form.ComboboxValue = "Help!";
+        Form.SliderValue = 70;
```

Comment:
What's going on with the commented out code here? Is that related to the next set of fixes?

Reply:
Uncommenting allows for testing of the initial value being set for the components